src/server/schema/resolvers/queries/listLives.ts

```diff
@@ -0,0 +1,10 @@
+import { getDatabaseContext } from '../../../database';
+import { GraphQLQueryResolvers } from '../definitions';
+
+const query: GraphQLQueryResolvers['listLives'] = async () => {
+    const { collections } = await getDatabaseContext();
+
+    return collections.lives.find().toArray();
+};
+
+export default query;
```

Comment:
EOF

Reply:
Fixed. Will include in next commit (part 5)